fusion_toolbox/Bfield.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+class Coil:
+    def __init__(self,pts,I):
+        """
+        Coil Class.
+
+        Parameters
+        ----------
+        pts : np.ndarray of shape (N,3)
+            List of x,y,z triplets which define a closed current loop. The last entry should be
+            equal to the first.
+        I : float
+            Coil current, in Amps
+        """
+        if not np.allclose(pts[0],pts[-1]): #If loop is not properly closed, connect first/last pts
+            pts = np.append(pts,pts[0][None,:],axis=0)
+        self.pts = pts
+        self.I = I
+
+    def B(self,xyz):
+        """
+        Returns the magnetic field 
+
+        Parameters
+        ----------
+        pts : np.ndarray with last dimension of size 3
```

Comment:
there is no parameters `pts`

Maybe you meant `xyz`
```suggestion
        xyz: np.ndarray with last dimension of size 3
```